controllers/datafoundationclient.go

```diff
@@ -208,3 +234,44 @@ func (r *dataFoundationClientReconciler) reconcileDefaultFSStorageClassClaim() e
 
 	return nil
 }
+
+func (r *dataFoundationClientReconciler) hasOCSVolumes() (bool, error) {
+	storageClassList := &storagev1.StorageClassList{}
+	if err := r.list(storageClassList); err != nil {
+		return false, fmt.Errorf("failed to list Storage Classes: %v", err)
+	}
+	ocsStorageClass := make(map[string]bool)
```

Comment:
No need to use make here
```suggestion
	ocsStorageClass := map[string]bool{}
```

Reply:
updated